### DISCUSSION TITLE
Local server deployment without docker

### DISCUSSION THREAD

**Discussion by jcurropena** [PrefectHQ/prefect#4229] [Q&A]
Hey everyone,

 we are validating prefect to be run in our production lab, which runs on Windows Server 2012, so we have no docker option here, though.

 Have anyone validated / performed on-premise installation of prefect and all the dependencies (Hasura, etc) running without docker-compose, and docker itself? Should we expect issues on prefect running on windows (Server)?

**Comment by zanieb** [comment]
We won't support this per https://github.com/PrefectHQ/server/issues/79. I'd say running the entire stack on Windows without docker sounds like a difficult option (it would definitely be feasible on linux). Perhaps someone else from the community will have tried it though.

**Reply by cicdw** [reply]
Yea, all agents should be Windows compatible so Cloud is always an option (which is much simpler to set up than Server in general), but otherwise I'd be surprised if this was possible.
